nautobot-app/{{ cookiecutter.project_slug }}/pyproject.toml

```diff
@@ -97,27 +98,27 @@ target-version = "py38"
 select = [
     "D",  # pydocstyle
     "F", "E", "W",  # flake8
-    "PL",  # pylint
+    "PL", # pylint
```

Comment:
We have been removing this as ruff does not use the named strings from `pylint: disable=`

Reply:
Josh ran a toml formatter here, I didn't want to clean this up. We have a separate issue to do post 2.3.0 cleanup (bandit, this rule etc)